src/mem.cpp

```diff
@@ -97,6 +98,7 @@ static int gds_map_gdr_memory(gds_mem_desc_t *desc, CUdeviceptr d_buf, size_t si
                 retcode = ret;
                 goto out;
         }
+        is_mh_created = 1;
```

Comment:
somewhere near gdr_open(), for v2, we want to add a one-off check for compatibility between the gdrcopy library used to build and the one used at run time